lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1413,7 +1415,13 @@ def configure_network(self, vm_obj):
                     nic_change_detected = True
 
             if nic_change_detected:
```

Comment:
`nic_change_detected` is always `True`.

Reply:
nic_change_detected is  set to False here https://github.com/ansible/ansible/blob/1dc8436ed91ef25748dd270c289c05f893cca6e3/lib/ansible/modules/cloud/vmware/vmware_guest.py#L1307 . Please can you check?

condition is executed if nic_change_detected: only if any network config is performed

Thanks

Reply:
Oh, sorry! My bad.